config/custom-environment-variables.yaml

```diff
@@ -64,8 +64,9 @@ datastore:
         prefix: DATASTORE_SEQUELIZE_PREFIX
 
 executor:
-    plugin: EXECUTOR_PLUGIN
```

Comment:
@stjohnjohnson's suggestion would give users much more control over the plugins that could be used, but we can refactor to support that later when people start writing their own executors.